bin/pull

```diff
@@ -71,10 +77,30 @@ done
 
 shift $((OPTIND-1))
 
+############# set session home directory
+# By default prepend points to last test session
+# input: requires $PR to be set
+# output: set $SESSION_DIR
+set_session_dir() {
+  count=$(find $PROG_HOME/site -name "$PR*" | wc -l)
+
+  if [[] $count == "0" ]]; then
+    SESSION_DIR="$PR"
+  elif [[ $FRESH == "true" ]]; then
+    SESSION_DIR="$PR-$(expr $count + 1)"
+  else
+    SESSION_DIR="$PR-$count"
+  fi
+}
+
 ############# Variables
 
 PR=$1
-FILE=$PROG_HOME/site/$PR/history.csv
+
+set_session_dir
+
+SESSION_HOME=$PROG_HOME/site/$SESSION_DIR
```

Comment:
Not specific to this PR, but usually in shell scripts it's a good idea to write "$FOO" instead of $FOO so that things don't break when there are spaces in the value of the variable.

Reply:
Thanks, that's indeed a good practice.